src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +36,33 @@
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{selectedTodo.id}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              onClick={() => onSelectedTodo(null)}
```

Comment:
The naming convention for event handlers could be improved. Instead of `onSelectedTodo`, consider using a name like `handleSelectTodo` or `onTodoSelect`. This makes it clear that this function is a handler for an event. Here is a link to a helpful article on naming conventions for event handlers: https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364